workitem/workitemtype.go

```diff
@@ -21,6 +21,8 @@ const (
 	SystemVersion = "version"
 
 	SystemRemoteItemID        = "system.remote_item_id"
+	SystemRemoteItemURL       = "system_remote_item_url"
+	SystemRemoteTrackerID     = "system_remote_tracker_id"
```

Comment:
@baijum This should be `SystemTrackerID`. 
We have remote item not remote tracker.

Reply:
The field was already added earlier #2366